sortdb/metadata.csv

```diff
@@ -0,0 +1,29 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration
+sortdb.stats.total_requests,GAUGE,,task,,Total number of requests,0,sortdb
+sortdb.stats.total_seeks,GAUGE,,task,,Total number of seeks,0,sortdb
+ sortdb.stats.get_requests ,GAUGE,,task,,Total number of get requests,0,sortdb
+ sortdb.stats.get_hits ,GAUGE,,task,,Total number of get hits,0,sortdb
+ sortdb.stats.get_misses ,GAUGE,,task,,Total number of get misses,-1,sortdb
+ sortdb.stats.get_requests.avg ,GAUGE,,task,,Average get requests received,0,sortdb

```

Comment:
This is in microseconds, as is the 95th and 99th percentiles.
https://github.com/jehiah/sortdb/blob/master/src/cmd/sortdb/http.go#L272

Addtionally, isn't lower better? (-1 instead of 0 in orientation)

Reply:
I am a bit confused about whether it is time or rate (per milliseconds) and whether we can actually say that low requests means better health of db (as per docs:  " 0 indicates no intrinsic preference in values. For example, rabbitmq.queue.messages or postgresql.rows_inserted where there is no preference for the size of the value or the preference will depend on the business objectives of the system. " ) But I can see what you are trying to say. @phil-hildebrand any thoughts?